src/linear_algebra/affine_space/affine_subspace.lean

```diff
@@ -1121,22 +1121,26 @@ begin
   { exact λ ⟨i₁, hi₁, hv⟩, ⟨p i₁, ⟨i₁, ⟨set.mem_univ _, hi₁⟩, rfl⟩, hv⟩ }
 end
 
-/-- The affine span of a set is nonempty if and only if that set
-is. -/
-lemma affine_span_nonempty (s : set P) :
-  (affine_span k s : set P).nonempty ↔ s.nonempty :=
+variables {s : set P}
```

Comment:
If you're going to introduce new variables, I'd prefer you wrap this handful of lemmas in a section

Reply:
Or preferably, just leave `{s : set P}` on each one since that makes the diff far smaller.